release-team/role-handbooks/release-team-lead/README.md

```diff
@@ -195,6 +203,11 @@ Coordinate with SIG-Release Chairs (who have access to the CNCF Service Desk as
   [kubernetes-release-team-shadows](https://groups.google.com/a/kubernetes.io/g/release-team-shadows)
   Google Group.
 - Ensure top-level OWNERS_ALIASES only includes Release Team personnel from four (4) releases, including the current one.
+  - Update `release-team-lead-role`, `ci-signal-role`, `enhancements-role`, `bug-triage-role`, `docs-role`,
+    `release-notes-role`, and `communications` role. These should include the relevant leads from all support releases
+    (as of the time of writing this is the last three) and the current in-development release.
+  - Update `release-team` to include the release leads for the all supporrted releases, the lead for the current
+    in-development release, and all SIG Release chairs and technical leads.
```

Comment:
not sure if this is needed; this might just lengthen this document and is very obvious when opening the OWNER_ALIASES file; WDYT